server/routerlicious/packages/services-core/src/combinedProducer.ts

```diff
@@ -12,7 +12,7 @@ export class CombinedProducer implements IProducer {
     constructor(private readonly producers: IProducer[]) {
     }
 
-    public async send(messages: object[], tenantId: string, documentId: string): Promise<any> {
+    public async send(messages: Record<string, unknown>[], tenantId: string, documentId: string): Promise<any> {
```

Comment:
Is this what typescript recommends now? Converting an object to Record<K, T> for better typing?

Reply:
Also this implements IProducer so I am wondering why TS is not complaining? Are objects and Record<string, unknown> interchangable?

Reply:
And there are more implementations of IProducer. So may be change them as well?

Reply:
This was a mistake. I experimented with some fixes and got sloppy about keeping them separate from the ESLint 7 upgrade itself. I'll remove these and replace with the disable comment. If we want to make this or a similar change we should do it separately from this PR.

Reply:
Resolved in an upcoming push.